### ISSUE TITLE
Jupyter Lab showing a 'Document is empty' display error after installing an extension

### ISSUE THREAD

**Issue by super-sebajin** [jupyterlab/jupyterlab#7189]
Hello there, I am running Jupyter Lab 1.0.2 on Chrome.  I have recently added the latex extension for jupyter lab and am now getting a display error:

![Screenshot (6)](https://user-images.githubusercontent.com/11996369/64702979-66e8ea00-d460-11e9-8835-c844d23d6e08.png)

Also, I get this when executing the browser's inspector:

![Screenshot (7)](https://user-images.githubusercontent.com/11996369/64703202-d1018f00-d460-11e9-9b07-1c6140ce936a.png)

I am new to Jupyter Lab but I do have working experience with Jupyter Notebooks. Hopefully this is an easily fixed issue. I  really do enjoy the Lab version more than the notebook.

**Comment by jasongrout** [comment]
This should be fixed as of a day ago or so. Can you try to clean and rebuild JupyterLab?

`jupyter lab clean`
`jupyter lab build`

This should have been fixed in #7182

**Comment by jasongrout** [comment]
If that doesn't work, can you give us set of instructions to reproduce the issue, starting with creating a new conda environment (if you are using conda)?

**Comment by telamonian** [comment]
> I reproduced this new issue with the filebrowser not showing anything with:
> 
> ```shell
> conda create -n del-jlabext jupyterlab=1.0.2
> jupyter labextension install @jupyterlab/latex
> jupyter lab
> ```
> 
> CC @telamonian, since it appears to be an icon-related issue.

@jasongrout I followed your recipe, and though I wasn't able to reproduce filebrowser not showing anything, I did reproduce the `NotebookTrustComponent` error message. I figured that one out. Before `1.1`, `@jupyterlab/statusbar` used to export a `IconItem` react component. `IconItem` was removed in favor of the icon handling stuff in `ui-components`. However, the code in `@jupyterlab/notebook@v1.0.2` still tries to import `IconItem` and use it as part of `NotebookTrustComponent`. So the definition of `IconItem` is `undefined`, which causes the error.

I have a sudden deep wish that I had gotten the new icon stuff in before `1.0`. The `IconItem` thing is fixable, but I still have no idea about the blank filebrowser
